src/webserver/Dynamic.jl

```diff
@@ -111,6 +111,7 @@ function notebook_to_js(notebook::Notebook)
                 "code" => cell.code,
                 "code_folded" => cell.code_folded,
                 "metadata" => cell.metadata,
+                "depends_on_skipped_cells" => cell.depends_on_skipped_cells,
```

Comment:
I think this makes more sense in `"cell_results"` like `"depends_on_disabled_cells"` meaning that you won't need to initialize it everywhere in the frontend.

Reply:
Yeah I didn't realize you would have to initialize it before going into the typescript error.
I put it into the cell_inputs as it seemed more appropriate there logically (I see it more as something related to the cell input rather than the cell result).
But indeed it's probably easier to make it part of the cell results

Reply:
> I see it more as something related to the cell input rather than the cell result

The rationale is that cell inputs are set by the user whereas cell results can be determined from all inputs (code, topology, etc...).

Reply:
That also makes sense! Just change the code accordingly